### ISSUE TITLE
Grey icon for "closed as not planned"

### ISSUE THREAD

**Issue by timhoffm** [octobox/octobox#3374]
### What feature you'd like to see
Github now distinguishes between successfully closed and "closed as not planned":

![grafik](https://github.com/octobox/octobox/assets/2836374/33dff8f4-74f3-4151-91bf-3e54b7382c82)

and uses different icons for that.

Both types are marked with the violet closed marker in octobox. It would be great if "closed as not planned" issues could get the grey icon in octobox as well.

## Motivation / Pitch

Making the closed reason visible via the icon is a usability bonus. It helps to quickly judge the state of the issue. Additionally, using the violet closed icon in octobox for issues that have the grey icon in GitHub is somewhat confusing. When seeing the violet icon. I consider it as completed because that's the color semantics I've learned from GitHub.

**Comment by andrew** [comment]
Yep this would be great to have, I wonder if the information is available in the REST API or will we have to switch to graphql to load it
